CONTRIBUTING.md

```diff
@@ -56,26 +56,29 @@ You can run danger locally to see what it would output by running the following
 swift run danger-swift pr [some-pr-url]
 ```
 
-### UI Tests (Mozillans/Pocketeers Only)
+### UI Tests (Mozillans Only)
 
-UI Tests are split into 3 test plans. As you built UI tests you should ensure that your test is enabled in 1 of the UI test plans to be run on CI in such a way as to balance the 3 test plans.
+UI Tests are split into 3 test plans. As you built UI tests you should ensure that your test is enabled in 1 of the UI test plans to be run on CI in such a way as to balance the 3 test plans. We use Snowplow-micro to test for expected analytics events. As a Mozillain you can get access to our Docker image.
 
 
 ### Localization
 
-Pocket localization is handled by Smartling. The project is currently setup in [Single Branch Mode](https://help.smartling.com/hc/en-us/articles/360008152513-GitHub-Connector-Overview#SingleBranchModeTranslationFlow) against the `develop` branch. Everytime a commit is made to `develop` Smartling will analyze the branch and determine if it needs to start a translation job. If it does, it will begin automatically and make a PR back against the repo with the needed translations.
+Pocket localization is handled by Smartling. The project is currently setup in [Single Branch Mode](https://help.smartling.com/hc/en-us/articles/360008152513-GitHub-Connector-Overview#SingleBranchModeTranslationFlow) against the `develop` branch. Everytime a commit is made to `develop` Smartling will analyze the branch and determine if it needs to start a translation job. 
+If it does, it will begin automatically and make a PR back against the repo with the needed translations.
```

Comment:
```suggestion
If it does, it will begin automatically and make a PR back against the repo with the needed translations.
Occasionally our translators may have a question or need some alterations to unblock their work. You can check in the Smartling Dashboard for these queries.
```